modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,98 @@ definition {
 		}
 	}
 
+	@description = "LPS-178783 - Verify that it's possible to send an e-mail notification with Action (Standalone)"
+	@priority = 5
+	test CanSendEmailViaActionStandalone {
+		property custom.properties = "feature.flag.LPS-173894=true";
+		property portal.acceptance = "true";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given an Email Notification Template and Custom Object with an Action (Standalone),") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "Liferay",
+				fromName = "Test Test",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "Subject");
+
+			PortletEntry.save();
+
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 178783",
+				objectName = "CustomObject178783",
+				pluralLabelName = "Custom Objects 178783");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject178783");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject178783");
+		}
+
+		task ("When the timing is set on Notification Configuration And entry is triggered manually") {
```

Comment:
Can we re-organize the steps here into the following? Creating the action should be part of the given.

1. Create the email notification template
2. Update the notification check interval settings
3. Create the custom object
4. Create the standalone action

So it'll look like this:

1. Given an Email Notification Template
2. And given the notification check interval set to 1
3. And given a custom object with a text field
4. And given a Standalone Action to send an email after an object entry is added